storage/innobase/xtrabackup/src/fil_cur.cc

```diff
@@ -143,9 +143,8 @@ xb_fil_cur_result_t xb_fil_cur_open(
   srv_close_files has an effect only on IBD tablespaces. */
   if (cursor->is_system || !srv_backup_mode ||
       (srv_close_files && cursor->is_ibd)) {
-    node->handle = os_file_create_simple_no_error_handling(
-        0, node->name, OS_FILE_OPEN, OS_FILE_READ_ONLY, srv_read_only_mode,
-        &success);
+    success = fil_node_open_file(node);
```

Comment:
We don't need success variable at all, we can check the return of fil_node_open_file directly on the if statement.

Reply:
sure, I will optimize this part